config/monitoring/alertmanager/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+labels:
+  - includeSelectors: true
+    pairs:
+      app.kubernetes.io/component: aerospike-alertmanager
```

Comment:
If we are defining this section, we don't have to add this label in all the resource yamls individually. 
It'll be easy to change from one place. 
Do this change for all the components wherever applicable